lib/src/network/network.spec.ts

```diff
@@ -2,9 +2,17 @@ import 'mocha';
 import { expect } from 'chai';
 
 import { Mainnet, Testnet, Devnet, fromChainId, NetworkEnum } from './network';
+import { FeeConfig, FeeAlgorithm } from '../fee';
```

Comment:
A side-track for `fee/types.ts`, could you append some comment for different config (normal & native one)? Besides, is the function `parseFeeConfigForNative` better put into `fee/index` instead of `fee/types`?

Reply:
`parseFeeConfigForNative` is intended for internal only. It is the types for calling the Rust function so does not need to be exposed to public.